src/foam/u2/wizard/WizardletSearchController.js

```diff
@@ -96,9 +96,17 @@ foam.CLASS({
           if ( seen.includes(cls) ) return;
           seen.push(cls);
           for ( let p of cls.getAxiomsByClass(foam.core.Property) ) {
-            var visibilityEnum = p.visibility instanceof Function
-              ? p.visibility.apply(w.data)
-              : p.visibility;
+            var visibilityEnum = '';
```

Comment:
think can lowkey just do
`var visibilityEnum = p.visibility`

And remove the else block on L107